skin.json

```diff
@@ -217,6 +217,14 @@
   "live": {
     "forceDvrDisabled": false
   },
+  "isVrAnimationEnabled": {
```

Comment:
How are we using this?  Usually we don't name a variable with a question. That makes me think it's a function.

Reply:
oh, I meant that values from this object have to be boolean. This values are used for showing animation message for video 360.

Reply:
It seems ok for now.